packages/camera/src/components/Controls/index.js

```diff
@@ -341,7 +341,7 @@ Controls.AddDamageButtonProps = {
 Controls.FullscreenButtonProps = {
```

Comment:
also please add a check in the Controls component to check if we are using Safari. If we are using Safari, we should add `visibility: 'hidden'` to the button with id `'full-screen'`